docs/usage/getting-started.md

```diff
@@ -41,6 +41,25 @@ options, run:
       headscale <COMMAND> --help
     ```
 
+## Manage headscale from another local user
```

Comment:
Can you move this into an admonition block instead of creating a new section? As it does not apply to all setups and should be a gentle note.